lib/ui/form/widget/form_survey_detail_page.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_survey/constants.dart';
+import 'package:flutter_survey/theme/app_dimensions.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class FormSurveyDetailPage extends StatelessWidget {
+  const FormSurveyDetailPage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        const BackButton(color: Colors.white),
+        const SizedBox(height: AppDimensions.spacing10),
+        Padding(
+          padding: const EdgeInsets.symmetric(
+            horizontal: AppDimensions.spacing20,
+          ),
+          // TODO: Integrate title from survey details #23
+          child: Text(
+            "Working from home Check-In",
+            style: Theme.of(context).textTheme.titleLarge,
+            maxLines: Constants.surveyTitleMaxLines,
```

Comment:
I don't think we should set maxlines for this widget as this one is the last chance to show the full question content to the user, not the same as the one in the Home screen page's item. 💭  In case of long content, we should make it scrollable instead.